OutOfSchool/OutOfSchool.BusinessLogic/Services/Elasticsearch/ElasticsearchSynchronizationService.cs

```diff
@@ -141,7 +141,7 @@ private async Task<bool> Synchronize(
             {
                 var result = await esProvider.DeleteRangeOfEntitiesByIdsAsync(ids).ConfigureAwait(false);
 
-                if (result == Result.Error)
+                if (result != Result.Deleted)
```

Comment:
why it is changed ?

Reply:
It has been changed because the Result enum in Elasticsearch 8.15 no longer contains an Error.